plans/k8s.tf

```diff
@@ -0,0 +1,201 @@
+resource "azurerm_resource_group" "k8s" {
+  name     = "${var.prefix}k8s"
+  location = "${var.location}"
+  tags {
+    env = "${var.prefix}"
+  }
+}
+
+resource "azurerm_storage_account" "k8s" {
+    name                = "${var.prefix}k8s"
+    resource_group_name = "${azurerm_resource_group.releases.name}"
+    location            = "${var.location}"
+    account_type        = "Standard_GRS"
+    tags {
+        env = "${var.prefix}"
+    }
+}
+
+resource "azurerm_template_deployment" "k8s"{
+  name  = "${var.prefix}k8s"
+  resource_group_name = "${ azurerm_resource_group.k8s.name }"
+  parameters = {
+        sshRSAPublicKey = "${file("~/.ssh/id_rsa.pub")}"
+        dnsNamePrefix = "${var.prefix}"
+# Bug with integer
+#        agentCount = 1
+#        masterCount = 1
+        agentVMSize = "Standard_A2"
```

Comment:
I think this should be a variable like `k8_agent_size` so production environments can be much bigger VMs